android/java/res/layout/brave_shields_third_layout.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) 2020 The Brave Authors. All rights reserved.
+     This Source Code Form is subject to the terms of the Mozilla Public
+     License, v. 2.0. If a copy of the MPL was not distributed with this file,
+     You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto"
+      xmlns:tools="http://schemas.android.com/tools"
+      android:orientation="vertical"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/shield_toggle_bg_color">
+
+    <View
+	    android:layout_width="match_parent"
+	    android:layout_height="1dp"
+	    android:alpha="0.5"
+	    android:background="@color/shield_divider_color"/>
+
+    <TextView
+        android:id="@+id/show_all_blocked_elements_text"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
+        android:layout_weight="1"
+        android:paddingHorizontal="16dp"
+        android:paddingTop="16dp"
```

Comment:
nit : we can use paddingVertical here as paddingTop and paddingBottom have the same values.